src/custom/state/claim/hooks/index.ts

```diff
@@ -749,37 +749,36 @@ export function useUserEnhancedClaimData(account: Account): EnhancedUserClaimDat
     const chainId = supportedChainId(preCheckChainId)
     if (!chainId) return []
 
-    return sorted.reduce<EnhancedUserClaimData[]>((acc, claim) => {
-      const tokenAndAmount = claimTypeToTokenAmount(claim.type, chainId)
-
-      if (!tokenAndAmount) return acc
-
-      const price = new Price({
-        baseAmount: ONE_VCOW,
-        quoteAmount: CurrencyAmount.fromRawAmount(tokenAndAmount.token, tokenAndAmount.amount),
-      }).invert()
-
-      // get the currency amount using the price base currency (remember price was inverted) and claim amount
-      const currencyAmount = CurrencyAmount.fromRawAmount(price.baseCurrency, claim.amount)
+    return sorted.map<EnhancedUserClaimData>((claim) => {
       const claimAmount = CurrencyAmount.fromRawAmount(ONE_VCOW.currency, claim.amount)
```

Comment:
Would it make sense, to make this a bit more readable, to move the logic of that enhances the claim data to a static function? 

Note that this is a logic inside a `useMemo`, inside a `map` and has some non trivial logic, probably can be factored out to a function which receives just the `claim` and the `chainId`

Reply:
Sure, I'll refactor it

Reply:
https://github.com/gnosis/cowswap/pull/2176